### PR TITLE
GT-Warmup-Cooldown

### PR DESCRIPTION
**Pull request by jkostiuk**
Warmups and Cooldowns for GT, further addressing #370. 

Meta note: I'm still open to suggestions for a different name other than cooldown. 

Let me know when/were I should be contributing some text to explain to instructors what these things are.

### CODE REVIEW

**Comment by jkostiuk** [comment]
hmmm..."looking ahead" implies an element of "preview" in the activities, which isn't necessarily true. 

How do you feel about "Checking In" or "Self-Assessment"? 

The main purpose of these activities is for students to see if they *really* have it, or need to do a little more review, etc.

**Comment by siwelwerd** [comment]
>Let me know when/were I should be contributing some text to explain to instructors what these things are.

I think Steven and I have loose plans to begin work on an instructor version during the Edit-A-Thon, in which case there will be a way to add a note at the beginning about this and anything else we want to communicate to instructors.

**Comment by StevenClontz** [comment]
> hmmm..."looking ahead" implies an element of "preview" in the activities, which isn't necessarily true.
> 
> How do you feel about "Checking In" or "Self-Assessment"?
> 
> The main purpose of these activities is for students to see if they _really_ have it, or need to do a little more review, etc.

Leaving this question open as it's discussed in #370

File: source/05-GT/02.ptx [hunk 1]
```diff
@@ -412,12 +462,6 @@ Based on the previous activities, which technique is easier for computing determ
 <m>\mbox{det}\left(\begin{array}{ccc}x&amp;y&amp;1\\x_1&amp;y_1&amp;1\\x_2&amp;y_2&amp;1\end{array}\right) = 0.</m>
                 </statement>
                 
-     </exploration>
-                 <exploration>
-                <statement> 
-                    Prove that the determinant of any diagonal matrix, upper triangular matrix, or lower triangular matrix, is the product of it's diagonal entries.
-                </statement>
-
```
Comment (StevenClontz) [inline source/05-GT/02.ptx:420]: Is this intentional? If so, let's discuss with @jford1906

**Review by StevenClontz — COMMENTED** [review]
I'm inclined to approve and merge given the structure of this content, without reviewing the pedagogical content as I trust @jkostiuk to use this in the fall and make appropriate adjustements.

However we need to address the removal of a writing activity from GT2 before merging.

File: source/05-GT/02.ptx (hunk 1 above)
Comment (StevenClontz) [reply source/05-GT/02.ptx:420]: In any case: typo `it's` -> `its`

**Comment by jford1906** [comment]
It's the breaking of the one writing activity into 3, right? I'm fine with
that.

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**